.github/workflows/build_and_test.yml

```diff
@@ -215,33 +219,18 @@ jobs:
             script: build-platform --linux-arm --skip-os-packages
     runs-on: ${{ matrix.os }}
     needs: build
-    defaults:
-      run:
-        working-directory: ./artifacts
```

Comment:
This is an artifact of my original implementation, which I simply forgot to take out before merging :P